### PR TITLE
chore: add missing return types and return statement

### PR DESCRIPTION
**Pull request by juantejer4**
# ⚡ Add missing return types to Laravel boilerplate stubs - [86dr3nyh6](https://app.clickup.com/t/86dr3nyh6) ⚡

## 💻 What type of change is this?

- [x] 💎 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Styling
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build / CI

## ⭐ Description

I've added return types  for all stub methods, and each including a return statement

## ✅ Checklist

- [x] This PR can be merged (it is not a draft, work in progress, or blocked on another PR)
- [ ] I have tested this change locally in multiple screen sizes
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: stubs/controller.stub [hunk 1]
```diff
@@ -2,44 +2,61 @@
...
+            ->respond();
     }
 
-    public function create()
+    public function create(): JsonResponse
     {
-        //
+        return responder()
+            ->success()
+            ->respond();
```
Comment (gianfranco-rocco) [inline stubs/controller.stub:25]:
```suggestion
        return View::make('create');
```

File: stubs/controller.stub [hunk 2]
```diff
@@ -2,44 +2,61 @@
...
     }
 
-    public function show(string $id)
+    public function show(string $id): View
     {
-        //
+        return responder()
+            ->success()
+            ->respond();
```
Comment (gianfranco-rocco) [inline stubs/controller.stub:39]:
```suggestion
        return View::make('show');
```

File: stubs/controller.stub [hunk 3]
```diff
@@ -2,44 +2,61 @@
...
     }
 
-    public function edit(string $id)
+    public function edit(string $id): JsonResponse
```
Comment (gianfranco-rocco) [inline stubs/controller.stub:42]:
```suggestion
    public function edit(string $id): ViewContract
```

File: stubs/controller.stub [hunk 4]
```diff
@@ -2,44 +2,61 @@
...
     {
-        //
+        return responder()
+            ->success()
+            ->respond();
```
Comment (gianfranco-rocco) [inline stubs/controller.stub:46]:
```suggestion
        return View::make('edit');
```

File: stubs/controller.stub [hunk 5]
```diff
@@ -2,44 +2,61 @@
...
     }
 
-    public function store(Request $request)
+    public function store(Request $request): JsonResponse
     {
-        //
+        return responder()
+            ->success()
+            ->respond();
```
Comment (gianfranco-rocco) [inline stubs/controller.stub:32]:
```suggestion
            ->respond(JsonResponse::HTTP_CREATED);
```

File: stubs/controller.stub [hunk 6]
```diff
@@ -2,44 +2,61 @@
...
     }
 
-    public function destroy(string $id)
+    public function destroy(string $id): JsonResponse
     {
-        //
+        return responder()
+            ->success()
+            ->respond();
```
Comment (gianfranco-rocco) [inline stubs/controller.stub:60]:
```suggestion
            ->respond(JsonResponse::HTTP_NO_CONTENT);
```

File: stubs/factory.stub
Comment (gianfranco-rocco) [inline stubs/factory.stub:1]: This stub's previous return statement was right. This is the stub for factories, not controllers. Changes should be reverted.

File: stubs/job.stub
Comment (gianfranco-rocco) [inline stubs/job.stub:1]: Revert changes.

File: stubs/mail.stub
Comment (gianfranco-rocco) [inline stubs/mail.stub:1]: Revert changes.

File: stubs/markdown-mail.stub
Comment (gianfranco-rocco) [inline stubs/markdown-mail.stub:1]: Revert changes.

File: stubs/markdown-notification.stub
Comment (gianfranco-rocco) [inline stubs/markdown-notification.stub:1]: Revert changes.

File: stubs/notification.stub
Comment (gianfranco-rocco) [inline stubs/notification.stub:1]: Revert changes.

Comment (gianfranco-rocco) [inline stubs/notification.stub:1]: Revert changes.

File: stubs/policy.stub
Comment (gianfranco-rocco) [inline stubs/policy.stub:1]: Revert changes.

File: stubs/request.stub
Comment (gianfranco-rocco) [inline stubs/request.stub:1]: Revert changes.

File: stubs/resource-collection.stub
Comment (gianfranco-rocco) [inline stubs/resource-collection.stub:1]: Revert changes.

File: stubs/resource.stub
Comment (gianfranco-rocco) [inline stubs/resource.stub:1]: Revert changes.

File: stubs/view-component.stub
Comment (gianfranco-rocco) [inline stubs/view-component.stub:1]: Revert changes.

**Review by gianfranco-rocco — CHANGES_REQUESTED** [review]

**Review by gianfranco-rocco — CHANGES_REQUESTED** [review]

**Comment by gianfranco-rocco** [comment]
@sojeda, I need your experience to correct this PR.

File: stubs/controller.nested.singleton.api.stub [hunk 7]
```diff
@@ -4,29 +4,35 @@ declare(strict_types=1);
...
-        //
+        return responder()
+            ->success(${{ parentModelVariable }})
+            ->respond();
     }
 
     public function destroy({{ parentModel }} ${{ parentModelVariable }}): never
     {
-        abort(404);
+        App::abort(JsonResponse::HTTP_NOT_FOUND);
```
Comment (sojeda) [inline stubs/controller.nested.singleton.api.stub:36]:
```
return responder()
            ->error()
            ->respond(JsonResponse::HTTP_NOT_FOUND);
```

**Pull request merged**